src/index.ts

```diff
@@ -91,6 +92,7 @@ export {
   getFlashloanWhitelist,
   fetchRegistryModules,
   getTrustAttestersAction,
+  findTrustAttesters,
```

Comment:
nit: can we name this `findTrustedAttesters` like the registry function?

Reply:
should it be renamed to `fetchTrustedAttersters`?